repositories/migrations/20250905163100_various_schema_improvements.sql

```diff
@@ -0,0 +1,120 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE decisions
```

Comment:
This is mostly a style / ease-to-read note, but I would merge settings that applies on the same table into a single statement. Knowing what applies to which table is a bit hard to figure out when they are all separate and spread out.

```
ALTER TABLE webhook_events
SET (
  autovacuum_analyze_scale_factor = 0.01,
  autovacuum_vacuum_insert_scale_factor = 0.02
);
```

The same applies to the `drop`s.